github/spec/features/validate_repos_spec.rb

```diff
@@ -0,0 +1,88 @@
+require_relative '../spec_helper'
+require_relative '../../lib/configure_repos'
+require_relative '../../lib/fetch_repos'
+require_relative '../../lib/validate_repos'
+
+require 'base64'
+require 'yaml'
+require "octokit"
+require "json"
+require "open-uri"
```

Comment:
We have a mixture of single and double quotes going on - we should really consider pulling in [rubocop-govuk](https://github.com/alphagov/rubocop-govuk/) into this repo so we can start to standardise this stuff. Maybe that's a PR you can open later, if you fancy? (Not part of this PR, as it's a separate thing, and let's not do it _before_ this PR as otherwise you'll have lots of conflicts to deal with).

But, I do have a question about these `require` calls - are they all needed? For example, we're mocking Octokit ourselves, so there should be no need to `require "octokit"` as far as I can tell.

Reply:
That's a great suggestion, I'll keep that in mind when we have finalised this one, thank you!